Example/IntegrationTests/Web3Wallet/Web3WalletClientTest.swift

```diff
@@ -0,0 +1,402 @@
+import XCTest
+import WalletConnectUtils
+import JSONRPC
+@testable import WalletConnectKMS
+@testable import WalletConnectSign
+@testable import Web3Wallet
+@testable import WalletConnectRelay
+import WalletConnectPairing
+import WalletConnectNetworking
+
+final class Web3WalletClientTests: XCTestCase {
```

Comment:
is it copy or Sign and Auth integration tests? Do we need this?

Reply:
to me it looks like a copy of just a sign tests.
running sign and web3wallet tests together will significantly increase testing on CI without many benefits.

what do you think to just unit test web3walletClient to ensure it calls sign and auth methods?

combining sign and auth tests into one class may be a bit monolithic approach 🤔

Reply:
it would require sign and auth clients mocks